### DISCUSSION TITLE
Question for the DAC audio board?

### DISCUSSION THREAD

**Discussion by banman11** [frntc/SIDKick-pico#1] [Q&A] [answered]
Hi Frenetic,

First of all Nice project!

Would something like this be suitable for the DAC audio board?

AU$2.18 | Interface I2S PCM5102A DAC Decoder GY-PCM5102 I2S Player Module pHAT Format Board Digital PCM5102 Audio Board
https://a.aliexpress.com/_mNhQgf2

Thank you.

**Comment by frntc** [comment, marked as answer]
Hi, thanks!

Your link didn't work for me, but the DAC-boards I used are of this type: https://de.aliexpress.com/item/1005005393398013.html

(there are many offers for the same board)

**Comment by banman11** [comment]
Hi Frenetic,

Your link is the identical board to the one I was looking at. Plus it's cheaper. Thank you.

I believe searching for GY-PCM5102 will bring up the suitable board with the pinouts in the correct orientation
